newIDE/app/src/EventsSheet/index.js

```diff
@@ -477,6 +497,10 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
             }
           }
         );
+
+        // This is not a real hook.
```

Comment:
is it not? Should this not be at the top of the sheet?

Reply:
It should in theory, but for now I'll let it here (I just made it closer to the place where it's used) to avoid any risk/regression (we are sure we're ready the type of the screen at the moment of the addition, and not an outdated value for some reason).